easybuild/tools/repository/filerepo.py

```diff
@@ -109,7 +109,11 @@ def add_easyconfig(self, cfg, name, version, stats, previous):
         else:
             statstxt = statscomment + statsprefix + stats_to_str(stats, isyeb=yeb_format) + statssuffix
 
-        txt += statstxt
+        try:
+            txt += statstxt
+        except UnicodeDecodeError as e:
```

Comment:
Style-wise we have a policy of not using single-letter variables outside of list comprehensions, so this should be `err` rather than `e`